app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+
+  validates :username, presence: true, uniqueness: true, length: { in: 3..10 }
+  validates :email, presence: true, uniqueness: true
```

Comment:
- [ ] Kindly validate the email has the propper format. There is a Regex that takes care of that.